src/Symfony/Bridge/Doctrine/DependencyInjection/CompilerPass/RegisterMappingsPass.php

```diff
@@ -111,7 +111,7 @@
      * @param string[]             $managerParameters       List of container parameters that could
      *                                                      hold the manager name.
      * @param string               $driverPattern           Pattern for the metadata driver service name
-     * @param string               $enabledParameter        Service container parameter that must be
+     * @param bool                 $enabledParameter        Service container parameter that must be
```

Comment:
@Tobion, false is not valid for this place in PHPdoc. In this place we must specify variable type but not specific value. I think correct is 'string|bool'.